shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/config/impl/StorageContainerConfigurationFactory.java

```diff
@@ -52,4 +54,29 @@ public static StorageContainerConfiguration newInstance(final DatabaseType datab
                 throw new RuntimeException(String.format("Database `%s` is unknown.", databaseType.getType()));
         }
     }
+    
+    /**
+     * Create new instance of storage container configuration.
+     *
+     * @param databaseType database type
+     * @param command command
+     * @param containerEnvironments container environments
+     * @param mountedResources mounted resources
+     * @return created instance
+     */
+    public static StorageContainerConfiguration newInstance(final DatabaseType databaseType, final String command, final Map<String, String> containerEnvironments,
+                                                            final Map<String, String> mountedResources) {
+        switch (databaseType.getType()) {
+            case "MySQL":
+                return MySQLContainerConfigurationFactory.newInstance(command, containerEnvironments, mountedResources);
+            case "PostgreSQL":
+                return PostgreSQLContainerConfigurationFactory.newInstance();
+            case "openGauss":
+                return OpenGaussContainerConfigurationFactory.newInstance();
+            case "H2":
+                return H2ContainerConfigurationFactory.newInstance();
+            default:
+                throw new RuntimeException(String.format("Database `%s` is unknown.", databaseType.getType()));
+        }
+    }
```

Comment:
It looks strange here, parameters are only used by MySQL type. Could we just create container configuration for MySQL directly? Use MySQLContainerConfigurationFactory